_std/defines/year.dm

```diff
@@ -0,0 +1 @@
+#define CURRENT_SPACE_YEAR 2053
```

Comment:
at least put this in a `time.dm`, jeez - maybe with day/night cycles?

Reply:
hrm. thats a really good idea!